Cargo.toml

```diff
@@ -34,3 +34,6 @@ features = [
 
 [target.'cfg(target_os = "redox")'.dependencies]
 redox_syscall = "0.2"
+
+[features]
+nightly = []
```

Comment:
How about using https://github.com/dtolnay/rustversion?

Reply:
Yeah fine by me if that's your preference.

Reply:
Actually..., never mind. Thinking through this, I'd rather someone _explicitly_ opt-in to this behavior if they need it rather than be surprised that something works on nightly but not elsewhere.